src/coreclr/vm/interpreter.cpp

```diff
@@ -12315,13 +12317,13 @@ void Interpreter::PrintValue(InterpreterType it, BYTE* valAddr)
 
     case CORINFO_TYPE_NATIVEINT:
         {
-            INT64 val = static_cast<INT64>(*reinterpret_cast<NativeInt*>(valAddr));
+            long long val = static_cast<long long>(*reinterpret_cast<NativeInt*>(valAddr));
```

Comment:
This isn't really correct. A `long long` is a minimum of 64-bit, but it can be more. This should be `int64_t` and then update the `ll` prefix following the "Format macro constants" table found at https://en.cppreference.com/w/cpp/types/integer.